### ISSUE TITLE
Illegal instruction

### ISSUE THREAD

**Issue by cikaldev** [Hax4us/TermuxAlpine#44]
## Device :
| Key | Value |
|:---|:---|
| OS | Android 7.1.1 armv7l |
| Host | samsung SM-J510FN |
| Kernel | 3.10.49-12476963 |
| Shell | bash 5.1.8 |
| CPU | Qualcomm MSM8916 (4) @ 1.190GHz |
| Memory | 1386MiB / 1891MiB |
| Termux | F-Droid 0.117 |

## Issue :
Today i follow instruction from **Readme** in this repo, installation success without any error. But after issued command `startalpine` i will try to install `nodejs`. What's wrong with my installation, can somebody tell me how to make it work ?

## Step to reproduce :
``` bash
$ apk update && apk upgrade
$ apk add ca-certificates git
$ apk add nodejs npm
$ which node
/usr/bin/node
$ node -v
Illegal instruction
$ which npm
/usr/bin/npm
$ npm -v
Illegal instruction
```

**Comment by ghost** [comment]
probably this node doesnt support arm. u can download node tarball for arm from https://nodejs.org/dist

**Comment by valium007** [comment]
@ghost, No! actually @cikaldev has a device with armv7l architecture, while the TermuxAlpine script downloads the armhf version
There is a very little difference between them

You need to download the armv7l version not the armhf
